js/interface.js

```diff
@@ -914,28 +908,33 @@ $(window).on('resize', windowResized).trigger('resize');
 $('#app')
   .on('click', '[data-order-date]', function() {
     var $dataSource = $(this);
+    var $defaultOrder = $(this).data('defaultOrder');
```

Comment:
defaultOrder is not a jQuery object.

```suggestion
    var defaultOrder = $(this).data('defaultOrder');
```

Reply:
Unresolved issue. Don't use `$` prefix when it's not a jQuery object.